packages/common/package.json

```diff
@@ -60,6 +61,7 @@
     "d3-ease": "1.0.5",
     "d3-format": "1.3.2",
     "d3-interpolate": "1.3.2",
+    "d3-hierarchy": "1.1.9",
```

Comment:
Should not be in common - the only d3 items in common are ones used by common...